outputs.tf

```diff
@@ -8,8 +8,8 @@ output "codebuild_project" {
 
 output "codebuild_role" {
```

Comment:
Usual  practice is to have separate outputs.
I think while we are here lets change this to:

codebuild_iam_role_name
And get rid of the ARN (unless we are using it somewhere).

Then instead of fallback on null, the name can fallback on the input variable.

If we need the arn then I suggest add another variable:
codebuild_iam_role_arn
And look up the ARN using a `data` if the input variable for IAM role was supplied